frontend/src/components/register/RegisterCard.tsx

```diff
@@ -60,17 +62,22 @@ const RegisterCard = () => {
       setErrors({ ...errors, email: "Email jest wymagany." });
     } else if (!re.test(String(value).toLowerCase())) {
       setErrors({ ...errors, email: "Niepoprawny adres email." });
+    } else if (value.length > 36) {
```

Comment:
jak zwiększymy długość emila to nie zapomnij zmienić też tutaj

Reply:
Ok, thx